ocaml/xapi/storage_impl.ml

```diff
@@ -172,10 +172,11 @@ module Vdi = struct
     ; dps: (Dp.t * Vdi_automaton.state) list
           (** state of the VDI from each dp's PoV *)
     ; leaked: Dp.t list  (** "leaked" dps *)
+    ; dpv: (Dp.t * Vm.t) list
```

Comment:
Some information in a doccoment would be welcome here
```suggestion
    ; dpv: (Dp.t * Vm.t) list  (** VM Datapath *)
```

Reply:
This needs a default added for the upgrade case: `[@default []]` at the end of the line.